docs/README-AWS.md

```diff
@@ -106,97 +146,121 @@ Resources:
         ElasticServerlessForwarderCloudWatchLogsEvents: ""  ## FILL WITH A COMMA DELIMITED LIST OF CLOUDWATCH LOGS LOG GROUPS ARNS TO SET SUBSCRIPTION FILTERS ON THE LAMBDA FOR (IF ANY).
         ElasticServerlessForwarderS3Buckets: ""             ## FILL WITH A COMMA DELIMITED LIST OF S3 BUCKETS ARNS THAT ARE THE SOURCES OF THE S3 SQS EVENT NOTIFICATIONS (IF ANY).
 ```
-* Deploy the Lambda from SAR running the following command:
-  * ```commandline
+1. Deploy the Lambda function from SAR by running the following command:
+
+   ```commandline
     aws cloudformation deploy --template-file sar-application.yaml --stack-name esf-cloudformation-deployment --capabilities CAPABILITY_IAM CAPABILITY_AUTO_EXPAND
     ```
```

Comment:
Same issue here with ordered lists and indentation. You have to indent lines 126-128 and 130-148 the same amount as 151-153 for a continuous list.

Right now it looks like this: 

<img width="708" alt="image" src="https://user-images.githubusercontent.com/10479155/176271588-89be35f7-c985-48cc-a0b2-019d35b028e7.png">